persistence_test.go

```diff
@@ -0,0 +1,54 @@
+package gocv
+
+import (
+	"testing"
+)
+
+func TestFileStorage(t *testing.T) {
+
+	fs := NewFileStorageWithParams("testdata/filestorage", FileStorageModeWrite|FileStorageModeFormatJson, "utf-8")
```

Comment:
Perhaps writing the file into a temp directory would be better and would pass on Windows?

Untested:

```go
pth := filepath.Join(os.TempDir(), "filestorage.json")
fs := NewFileStorageWithParams(pth, FileStorageModeWrite|FileStorageModeFormatJson, "utf-8")
```

Reply:
change done